readme.md

```diff
@@ -142,6 +142,16 @@ try {
 }
 ```
 
+## Browser support
```

Comment:
Documenting it to be pro-active since users are most likely going to open issues with some unique build configuration where this might not work. :)